### PR TITLE
add trim modifier

### PR DESCRIPTION
**Pull request by kazupon**

### CODE REVIEW

File: src/guide/forms.md [hunk 1]
```diff
@@ -344,3 +344,11 @@ If you want user input to be automatically typecast as a number, you can add the
 ```
 
 This is often useful, because even with `type="number"`, the value of HTML input elements always returns a string.
+
+### `.trim`
+
+If you want user input to be automatically trim, tou can add the `trim` modifier to youre `v-model` managed inputs:
```
Comment (LinusBorg) [inline src/guide/forms.md]: >If you want user input to be automatically trim

*If you want user input to be trimmed automatically...*

Comment (kazupon) [inline src/guide/forms.md]: Thank you for your review!
sorry, poor my english 😿

File: src/guide/forms.md [hunk 2]
```diff
@@ -347,7 +347,7 @@ This is often useful, because even with `type="number"`, the value of HTML input
 
 ### `.trim`
 
-If you want user input to be automatically trim, tou can add the `trim` modifier to youre `v-model` managed inputs:
+If you want user input to trimmed automatically, you can add the `trim` modifier to youre `v-model` managed inputs:
```
Comment (LinusBorg) [inline src/guide/forms.md]: Sorry to bother you with this again, but there is still a grammar error:

> If you want user input to **be** trimmed automatically,

**be** is missing.

Comment (kazupon) [inline src/guide/forms.md]: oops, I fixed now.

File: src/guide/forms.md [hunk 3]
```diff
@@ -347,7 +347,7 @@ This is often useful, because even with `type="number"`, the value of HTML input
 
 ### `.trim`
 
-If you want user input to trimmed automatically, you can add the `trim` modifier to youre `v-model` managed inputs:
+If you want user input to be trimmed automatically, you can add the `trim` modifier to youre `v-model` managed inputs:
```
Comment (chrisvfritz) [inline src/guide/forms.md]: Just one more fix. 😃  `youre` should be `your`.

Comment (kazupon) [inline src/guide/forms.md]: oooops,  😹 
fixed now.

**Pull request merged**